iowrite.cpp

```diff
@@ -1358,6 +1365,16 @@ bool writeRestart(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
    restartReducer.addOperator(new DRO::MPIrank);
    restartReducer.addOperator(new DRO::BoundaryType);
    restartReducer.addOperator(new DRO::BoundaryLayer);
+   // Needed for re-adapting the mesh and checking memory use
+   restartReducer.addOperator(new DRO::DataReductionOperatorCellParams("vg_alpha",CellParams::ALPHA,1));
+   // Consider removing these later and just outputting them if needed
+   restartReducer.addOperator(new DRO::DataReductionOperatorCellParams("vg_reflevel",CellParams::REFINEMENT_LEVEL,1));
+   restartReducer.addOperator(new DRO::DataReductionOperatorCellParams("vg_x",CellParams::XCRD,1));
+   restartReducer.addOperator(new DRO::DataReductionOperatorCellParams("vg_y",CellParams::YCRD,1));
+   restartReducer.addOperator(new DRO::DataReductionOperatorCellParams("vg_z",CellParams::ZCRD,1));
```

Comment:
Are these really required? The visit plugin manages to calculate the cell coordinates from the MESH_NODE_CRDS array and the refinement level (which it obtains from the cellID). Not storing these values for every cell would save quite a bit of storage.

Reply:
Spatial coordinates are used here so the estimated refinement can be plotted in Python, as the ionosphere is never refined or unrefined. Blocks are used to estimate the memory and node usage. They could be both removed here, but in my opinion it makes sense that you could calculate these estimates from the restart file instead of the nearest regular bulk file.

Reply:
In practice we save bulk files every 0.5 or 1 s, dynamics in the runs are not that fast physically, so I think we can live with the slight inaccuracy and save on the restart size.